core/src/main/java/com/scalar/db/common/TableMetadataManager.java

```diff
@@ -21,7 +21,7 @@ public class TableMetadataManager {
 
   public TableMetadataManager(Admin admin, long cacheExpirationTimeSecs) {
     CacheBuilder<Object, Object> builder = CacheBuilder.newBuilder();
-    if (cacheExpirationTimeSecs > 0) {
+    if (cacheExpirationTimeSecs >= 0) {
```

Comment:
After this change, we can completely disable the metadata cache when setting `cacheExpirationTimeSecs` to `0`.

Reply:
Without this change, when `cacheExpirationTimeSecs=0`, the `CacheLoader` would still have a default `expireAfterWrite` value ?

Reply:
If `cacheExpirationTimeSecs=0` without this change, as we don't set `expireAfterWrite` value, the `CacheLoader` doesn't expire any cache entries.

So after this change,
- if we set `cacheExpirationTimeSecs` to a negative value, enabling cache but no expiration
- if we set `cacheExpirationTimeSecs` to `0`, disabling cache
- if we set `cacheExpirationTimeSecs` to a positive value, enabling cache and expiration

Reply:
Ok, thank your for the clarification. 👍